src/shared/components/ncUserAndGroupPicker/NcUserAndGroupPicker.vue

```diff
@@ -0,0 +1,186 @@
+<template>
+	<div class="row space-B">
+		<h3>{{ t('tables', 'Transfer this table to another user') }}</h3>
```

Comment:
Can we move this part out to the TransferTable component? Then the NcUseraAndGroupPicker is more generic and could be reused in other places